analiza-rozwiazan-biblioteki-komponentow-GUI.rst

```diff
@@ -0,0 +1,330 @@
+      RWD)
+   3. Open source (otwarta licencja bez ograniczeń)
+   4. Popularność (wśród społeczności programistycznej)
+   5. Aktywność społeczności na GitHub
+   6. Deklarowana dostępność pod kątem osób niepełnosprawnych
+   7. Kompatybilność z przeglądarkami
+   8. Zasoby komponentów
+   9. Gridy i layouty (sposób organizacji siatki, na której oparty jest
+      interfejs graficzny)
+
+2. poziomie funkcjonalnym – opracowano analizę istniejących rozwiązań w
+   administracji publicznej:
+
+   1. Wielka Brytania – Gov.uk https://www.gov.uk/service-manual
+   2. USA – 18F https://standards.usa.gov/design-principles/
+   3. Australia – Digital Transformation DTA
+      https://www.dta.gov.au/standard/
+
+Konkluzja
+~~~~~~~~~
+
+-  Po przeprowadzeniu analizy 16 frameworków frontendowych ograniczono
+   ich wybór do 3 frameworków, które uznano za najbardziej efektywne w
+   kontekście strategii użycia w platformie aplikacje.gov.pl: Bootstrap,
+   Foundation, Lightning Design System (http://getbootstrap.com/,
+   http://foundation.zurb.com/, https://lightningdesignsystem.com)
+
+   -  Kolejnym krokiem będzie przeprowadzenie eksperymentu
+      programistycznego. Będzie on polegał na wdrożeniu wybranych
+      ekranów makiety aplikacji EZD z użyciem 3 powyższych narzędzi.
+      Celem eksperymentu będzie sprawdzenie czasu wdrożenia, wygody
+      korzystania oraz jakości zakodowanego produktu.
+   -  Efektem eksperymentu będzie decyzja strategiczna o wsparciu
+      jednego lub więcej frameworków. Poprzez wsparcie należy rozumieć
+      stworzenie nakładek styli, które będą utrzymywane w zgodności z
+      aktualną wersją frameworku. Nakładki styli to zbiór reguł
+      opisujących wygląd i zachowanie poszczególnych elementów GUI –
+      będą obejmowały zarówno dokumentację i wytyczne, jak i kod. Plusem
+      takiego rozwiązania jest uwolnienie wykonawcy aplikacji od wyboru
+      konkretnego frameworku frontendowego i jednocześnie zachowanie
+      spójności interfejsu.
+
+-  Powstała koncepcja Standardów Projektowania (wieloaspektowa wizja
+   rozwoju aplikacji dla administracji publicznej.
+-  Została wypracowana lista komponentów biblioteki GUI w kontekście
+   prac urzędnika w systemie EZD.
+
+Analiza istniejących frameworków frontendowych
+----------------------------------------------
+
+Wybrano 16 popularnych frameworków frontendowych (CSS). Frameworki te są
+stale rozwijane, mają aktywną społeczność, są open source’owe i
+wspierają technologię RWD.
+
+Początkowy wybór frameworków objął:
+
+http://getbootstrap.com/ http://foundation.zurb.com/
+https://semantic-ui.com/ https://getmdl.io/ http://materializecss.com/
+https://purecss.io/ https://material.angularjs.org/latest/
+http://getskeleton.com/ https://ant.design/ https://getuikit.com/
+https://milligram.github.io/ http://susy.oddbird.net/
+https://material.io/ https://daneden.github.io/Toast/
+https://lightningdesignsystem.com https://imperavi.com/kube/
```

Comment:
Dlaczego to zrobiono tak nieczytelnie w dwóch kolumnach? Dlaczego nie są wymienione tytuły, a tylko spis linków?